src/main/scala/com/fulcrumgenomics/sopt/cmdline/ClpArgumentDefinitionPrinting.scala

```diff
@@ -55,7 +57,8 @@ object ClpArgumentDefinitionPrinting {
       argumentDefinition.shortName,
       argumentDefinition.typeDescription,
       makeCollectionArity(argumentDefinition),
-      makeArgumentDescription(argumentDefinition, argumentLookup))
+      makeArgumentDescription(argumentDefinition, argumentLookup),
+      argumentDefinition.optional)
```

Comment:
Could this be hoisted back onto the prior line please.

Reply:
It can, but why should it?  It's not an argument to `makeArgumentDescription`

Reply:
Nevermind, I misread.